src/occupation.jl

```diff
@@ -11,15 +29,12 @@ function compute_occupation(basis::PlaneWaveBasis{T}, eigenvalues, εF;
     inverse_temperature = iszero(temperature) ? T(Inf) : 1/temperature
 
     filled_occ = filled_occupation(basis.model)
-    [filled_occ * Smearing.occupation.(smearing, (εk .- εF) .* inverse_temperature)
-     for εk in eigenvalues]
+    occupation = [filled_occ * Smearing.occupation.(smearing, (εk .- εF) .* inverse_temperature)
+                  for εk in eigenvalues]
+    occupation
```

Comment:
Why? The function is called `compute_occupation` ... sort of clear what it returns, no?